renderdoc/driver/vulkan/vk_pixelhistory.cpp

```diff
@@ -86,7 +88,7 @@ struct EventInfo
   uint8_t original_stencil[8];
 };
 
-#if 1
+#if 0
```

Comment:
Accidentally left this change in :smile:.

Reply:
I actually left it on purpose, to get all the presubmit checks results.. I was going to modify after addressing all review comments, and passing the presubmit. Hope that works.

Reply:
Ah, that makes sense. In future I'll only mention this once things are otherwise ready to merge.